pre-commit

```diff
@@ -79,3 +79,5 @@ cargo test || error "Test failures"
 cargo test --features shuttle || error "Concurrency test failures"
 
 cargo bench --bench oneshot_ipa --no-default-features --features="enable-benches" || error "Failed to run IPA benchmark"
+
+cargo bench --bench oneshot_sort --no-default-features --features="enable-benches" || error "Failed to run IPA benchmark"
```

Comment:
Missing end of line here.

Also, can we just run very, very small tests in the pre-commit?  This is a pretty large batch.